### ISSUE TITLE
Resizing a ListView causes funky text resizing

### ISSUE THREAD

**Issue by j9ac9k** [ColinDuquesnoy/QDarkStyleSheet#124]
### Describe Your Environment

[Versions from your environment]

- QDarkStyle: 2.6.5
- OS: macOS 10.14
- Python: 3.7.1

[If used, please inform their versions]

- PySide: 5.11.2
- PyQt:
- PyQtGraph: dev
- QtPy: 1.5.2
- QT_API: pyside2
- PYQTGRAPH_QT_LIB:

### Language

Python

### Description / Steps to Reproduce [if necessary]

[Description of the issue]

1. Create a QMainWindow with a QSplitter, on the left widget have a QListView and on the right frame place a QFrame
2. Resize the splitter such that the text in the QListView entries is cut off
3. Some of the text of the QListView will resize

### Actual Result

![out](https://user-images.githubusercontent.com/646398/49976885-ced1ca80-fef8-11e8-97a0-597feb217e1d.gif)

### Expected Results / Proposed Result
Here is the same operation, but with the the following lines commented out

```python
...
import qdarksheet
...
self.setStyleSheet(qdarkstyle.load_stylesheet_from_environment())
...
```

![out_correct](https://user-images.githubusercontent.com/646398/49976898-d7c29c00-fef8-11e8-9b20-0996dca25926.gif)

### Relevant Code [if necessary]

I will happily provide code, current code is pretty split up, but if it would help for me to provide a minimum example, let me know and I'll hack one together.

**Comment by dpizetta** [comment]
Well, .... it gets bigger when it enters to scroll... uahauh...let me check here...may I'll as you for the code :)

![source](https://user-images.githubusercontent.com/5084939/49977254-be3b4780-ff2c-11e8-84c4-765f67a16fb3.gif)

**Comment by j9ac9k** [comment]
I may not have shown it well, but only _some_ of the elements get larger, some of the elements that are a bit shorter (like the top 10) never actually change in size.  

Let me know if you would like some specific code, but the structure is `QMainWindow`, with a `QSplitter` as the central widget.  The `QSplitter` has a populated `QListView` on the left, and an empty `QFrame` on the right.

**Comment by j9ac9k** [comment]
Just updated to PySide2 5.12, can confirm the issue is present there as well

I should also point out that this issue isn't consistent, it occurs about half the time I run the app (which is what make diagnosing the problem take so long, I had to re-run my application enough times to be convinced it was a qdarkstylesheet issue).  If anyone finds a way to consistently bring out this problem, be sure to post it!

**Comment by dpizetta** [comment]
@j9ac9k  any update about this issue? New versions of PySide/Pyqt also has it?Tks

**Comment by j9ac9k** [comment]
Hi @dpizetta I haven't looked at it as PySide2 5.12 supports recognizing when the OS is set to dark mode and adjusts accordingly, and as a result I haven't used the qdarkstylesheet.

That said, as I created the issue, the least I can do is incorporate the qdarkstylesheet and try again and see what happens (I also pinned my PySide2 to 5.12.0 for other reasons, but this is a good reason for me to try upgrading to 5.12.2)

**Comment by dpizetta** [comment]
Okay. If you have time to test it would be nice. Otherwise, no problems at all :) Tks

**Comment by j9ac9k** [comment]
I incorporated qdarkstylesheet in my project in its current state and could _not_ reproduce this issue I reported earlier (on pyside 5.12.2).  I attempted to go back in history of my git repo and reconstruct the project with only pyside 5.12.2 enabled; but had problems rebuilding my project.

Regardless, I'm good with closing out this issue.  Sorry I was unable to definitively point the finger at PySide 5.12.0 vs. 5.12.2 as a fix.
